projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -56,22 +58,43 @@ export default function SubscriptionControls( {
 } ) {
 	return (
 		<>
+			{ subscriberCount > 1 && (
+				<div
+					style={ {
+						margin: 20,
+						padding: 24,
+						background: '#f2f2f2',
+						borderRadius: '4px',
+						display: 'flex',
+						alignItems: 'center',
+						justifyContent: 'space-between',
```

Comment:
Is there some class we could leverage from Core, to ensure this remains relevant in the future?

Reply:
No, but it's something that should make it into the Jetpack component set, since we plan to use it in the publish flow as well.

Reply:
Moved to a `<InspectorNotice>` component. https://github.com/Automattic/jetpack/pull/22694/commits/4c794302f128f891b52731cacf0cdcf75bbc45f6